app/assets/stylesheets/views/_homepage.scss

```diff
@@ -198,11 +199,8 @@ body.homepage {
 
 .home-promo__link {
   @include govuk-font(24, $weight: bold);
+  @include govuk-responsive-margin(2, "bottom");
```

Comment:
These links look very close together on mobile, is it worth taking the opportunity to space them apart a little? The total height of the links and their parent `li`s appears to be only 26px.

Reply:
I'm not sure I follow this one though – I can't see any `li` elements in the promo section. Would you be able to expand on this?

Reply:
Ah yes sorry I've got totally confused here. I thought this change applied to the links in the black box, but I see now it doesn't. Ignore me.

(although those links are pretty close together, something for someone to address at some point perhaps)